content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -1586,22 +2366,46 @@ The name of the `name` argument on the `@relation` attribute can be omitted (`re
 @relation(_ name: String?, fields: FieldReference[]?, references: FieldReference[]?)
 ```
 
+#### Examples
+
+See: [The `@relation` attribute](../../concepts/components/prisma-schema/relations#the-relation-attribute).
+
 ### <inlinecode>@map</inlinecode>
 
-Maps a field name or enum value from the Prisma schema to a column with a different name in the database. If you do not use `@map`, the field name matches the column name exactly.
+Maps a field name or enum value from the Prisma schema to a column or field with a different name in the database. If you do not use `@map`, the field name matches the column name exactly.
```

Comment:
columns are called fields for MongoDB? that makes it super confusing vs. Prisma schema fields :(

Reply:
We can write document field maybe.

Reply:
```
Maps a field name or enum value from the Prisma schema to a column or document field with a different name in the database. If you do not use `@map`, the Prisma field name matches the column name or document field name exactly.
```